### PR TITLE
add male_student flag

### PR DESCRIPTION
**Pull request by lislis**
Hey there!
As we had the this discussion some time ago, I'll just did a simple implementation of an indicator if a group welcomes men as students.
Please let me now what you think.
I also added some explanations about our other flags.
screen shot 2016-03-28 at 00 56 52
screen shot 2016-03-28 at 00 56 42

### CODE REVIEW

File: app/views/groups/_form.html.haml [hunk 1]
```diff
@@ -64,5 +64,9 @@
       .form-group
         = f.check_box :inactive
         = f.label(:inactive, "Is your group inactive?")
+      .form-group
+        = f.check_box :allow_male_students
+        = f.label(:allow_male_students, "Do you welcome male students?")
+        %p.help-block Most groups prefer to only have students who indentify as women. Please indicate if you also welcome men.
```
Comment (berlintam) [inline app/views/groups/_form.html.haml]: perhaps instead "Please indicate if you also welcome students who identify as men"

File: db/schema.rb [hunk 2]
```diff
@@ -11,9 +11,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160105185150) do
+ActiveRecord::Schema.define(version: 20160327223742) do
 
-  create_table "friendly_id_slugs", force: true do |t|
+  create_table "friendly_id_slugs", force: :cascade do |t|
```
Comment (berlintam) [inline db/schema.rb]: I guess this change was generated on its own. Any idea what it does & why?

**Comment by berlintam** [comment]
woot woot! love it!

File: app/views/groups/_form.html.haml (hunk 1 above)
Comment (berlintam) [inline app/views/groups/_form.html.haml]: Also maybe 'many groups' instead of 'most groups'.

File: db/schema.rb (hunk 2 above)
Comment (lislis) [inline db/schema.rb]: actually, no. I'll reset that

**Comment by lislis** [comment]
I made the changes and rebased!

**Comment by lislis** [comment]
I did have to run the migrations that changed the that one line again that @berlintam commented on earlier. But no idea what it does.

File: app/views/groups/_form.html.haml [hunk 3]
```diff
@@ -64,5 +64,9 @@
       .form-group
         = f.check_box :inactive
         = f.label(:inactive, "Is your group inactive?")
+      .form-group
+        = f.check_box :allow_male_students
+        = f.label(:allow_male_students, "Do you welcome male students?")
+        %p.help-block Many groups prefer to only have students who indentify as women. Please indicate if you also welcome students who identify as men.
```
Comment (til) [inline app/views/groups/_form.html.haml]: "indentify" *chuckle*

**Comment by til** [comment]
LGTM expect for the unrelated change in schema.rb.

**Comment by lislis** [comment]
@til could it be that the change in the schema is related to some newer version of something? When I run the migrations it just happens?

**Comment by til** [comment]
Yes, apparantly it is caused by an update of rails:

http://stackoverflow.com/questions/27697119/what-is-cascade-in-rails-schema-rb-and-where-did-it-come-from

I think your version, the one with :cascade, is the newer one, and since
this is the first time that schema.rb is changed since the rails update,
this gets updated as well.

So it is ok to update it I'd say in this case. Usually I would advise to
put changes like that in a separate commit, because it is not related to
the functionality that you are adding here, but I don't see how this
would be possible in this case, because it is only triggered when the
schema is changed.

**Comment by lislis** [comment]
I separated the migration and schema into its own commit. I can't really pull them apart any further. I also rebased. Please merge!

**Comment by berlintam** [comment]
https://media.giphy.com/media/Q1D81ZTkMxQ1W/giphy.gif

**Comment by lislis** [comment]
nod

**Pull request merged**